Demo/src/main/java/com/braintreepayments/demo/PayPalMessagingFragment.kt

```diff
@@ -35,17 +38,17 @@ class PayPalMessagingFragment : BaseFragment(), PayPalMessagingListener {
             color = PayPalMessagingColor.BLACK
         )
 
-        // TODO: Refactor BraintreeClient out of PayPalMessagingView
-//        val payPalMessagingView = PayPalMessagingView(braintreeClient, requireActivity())
-//        payPalMessagingView.setListener(this)
-//        payPalMessagingView.start(payPalMessagingRequest)
-//        payPalMessagingView.layoutParams = ViewGroup.LayoutParams(
-//            ViewGroup.LayoutParams.MATCH_PARENT,
-//            ViewGroup.LayoutParams.MATCH_PARENT
-//        )
-//
-//        val messagingView: LinearLayout = view.findViewById(R.id.content)
-//        messagingView.addView(payPalMessagingView)
+        val braintreeClient = BraintreeClient(requireContext(), authStringArg)
```

Comment:
It can be a separate TODO but I think we want to encapsulate the `BraintreeClient` within the `PayPalMessagingView` - we made that change with other feature clients across v5 (merchants no longer need to be aware of `BraintreeClient` at all